core/utils.py

```diff
@@ -0,0 +1,27 @@
+import jwt
+
+from functools import wraps
+from django.http import JsonResponse
+from django.conf import settings
+
+from users.models import User
+
+def access_check(func):
```

Comment:
사소할 수는 있는데, 보통 함수이름 만들 때 동사형을 먼저 작성합니다.
```suggestion
def check_access(func):
```

Reply:
수정했습니다 감사합니다~